src/path-optimization/simple-time-parameterization.cc

```diff
@@ -150,7 +164,7 @@ namespace hpp {
         const DevicePtr_t& robot = problem().robot();
         vector_t ub ( robot->model().velocityLimit),
                  lb (-robot->model().velocityLimit),
-                 cb ((ub + lb) / 2);
+                 cb (vector_t::Zero(ub.size()));//cb ((ub + lb) / 2);
```

Comment:
I do not understand this modification. cb is not initialzed anymore ?

Reply:
It is expected to be 0 because lb = - ub.

Reply:
I wrote that having in mind that the lower and upper velocity bounds are different. At the moment, this is not possible.

Reply:
Then, I have to change my test in https://github.com/humanoid-path-planner/hpp-core/pull/204, or should I simply remove the PR ?

Reply:
Or keep only the additional assertions ?